qubesbuilder/plugins/source_deb/scripts/clamp-changelog-entry-date

```diff
@@ -30,9 +33,10 @@ if [ ! -r "$CHANGELOG_PATH" ]; then
     exit 1
 fi
 
-# get previous date
-PREVIOUS_DATE=$(grep '^ --' "$CHANGELOG_PATH" | head -n 2 | tail -n 1 | grep -o '  .*')
+# If PREVIOUS_DATE is not needed, remove this line:
+# PREVIOUS_DATE=$(grep '^ --' "$CHANGELOG_PATH" | head -n 2 | tail -n 1 | grep -o '  .*')
+
+# Replace the topmost date with CURRENT_DATE
+sed -e "0,/^ --/s/^\( --.*\)\(  .*\)/\1$CURRENT_DATE/" -i "$CHANGELOG_PATH"
```

Comment:
CURRENT_DATE is not set

I think what you need is to set CURRENT_DATE like this:
```
if [ -n "$SOURCE_DATE_EPOCH" ]; then
    CURRENT_DATE=$(date -R -u -d "@$SOURCE_DATE_EPOCH")
else
    CURRENT_DATE=$(grep '^ --' "$CHANGELOG_PATH" | head -n 2 | tail -n 1 | grep -o '  .*')
fi